router/batchrouter/handle_lifecycle.go

```diff
@@ -210,6 +210,8 @@ func (brt *Handle) setupReloadableVars() {
 	brt.warehouseServiceMaxRetryTime = config.GetReloadableDurationVar(3, time.Hour, "BatchRouter.warehouseServiceMaxRetryTime", "BatchRouter.warehouseServiceMaxRetryTimeinHr")
 	brt.datePrefixOverride = config.GetReloadableStringVar("", "BatchRouter.datePrefixOverride")
 	brt.customDatePrefix = config.GetReloadableStringVar("", "BatchRouter.customDatePrefix")
+	brt.useAWSV2 = config.GetReloadableBoolVar(false, "BatchRouter.useAWSV2")
```

Comment:
Shall we keep the same config option for all aspects of this change, e.g. `FileManager.useAwsSdkV2`?

Reply:
Yeah , I will clean up things once go-kit PR gets merged. Will make sure there is only one flag to enable and disable the v2 SDK